setup.cfg

```diff
@@ -13,7 +13,7 @@ use_parentheses=True
 line_length=100
 
 [green]
-processes     = 1
+#processes     = 1
```

Comment:
What about removing it at all? Or why it have to stay?

Reply:
Er, typo?